scheme.sql

```diff
@@ -0,0 +1,100 @@
+CREATE DATABASE taskforce CHARACTER SET utf8 COLLATE utf8_general_ci;
+
+USE taskforce;
+
+CREATE TABLE users (
```

Comment:
```suggestion
CREATE TABLE user (
```
Сущности бд и в коде принято называть в ед.числе